string tourName
string tourLevel
---
bool result